packages/bundle-size/src/utils/collectLocalReport.test.js

```diff
@@ -0,0 +1,89 @@
+const fs = require('fs').promises;
+const process = require('process');
+const tmp = require('tmp');
```

Comment:
To ensure things are cleaned up you need to add this line once:
```js
tmp.setGracefulCleanup();
```
And this option on every directory creation:
```js
tmp.dirSync('whatever', { unsafeCleanup: true });
```

Otherwise it won't delete folders on exit if they still contain files (which IMO is a bit silly, but oh well).

Reply:
@ecraig12345 updated as you suggested in 0ce884e. Can you please check? 🙏